Useful tools & utilities/Libraries.md

```diff
@@ -55,29 +49,23 @@
 
 	More info with code examples is available [here](http://square.github.io/okhttp/).
 
-5. **Chuck**
+5. **Chucker**
 
-	Chuck is a network library that logs the requests and responses during runtime on the device itself. This library should be enabled on staging and/or debug builds and should **never** be used in production. This is a required library that should always be implemented in the aforementioned environments.
+	Chucker is a network library that logs the requests and responses during runtime on the device itself. This library should be enabled on staging and/or debug builds and should **never** be used in production. This is a required library that should always be implemented in the aforementioned environments.
 
-	More info with code examples is available [here](https://github.com/jgilfelt/chuck).
+	More info with code examples is available [here](https://github.com/ChuckerTeam/chucker).
 
 ## Database
 
-1. **DBFlow**
-
-	DBFlow is an ORM Android database library with annotation processing. It is built on top of the SQLite database and has support for all SQL actions and transactions.
-
-	More info with code examples is available [here](https://github.com/Raizlabs/DBFlow).
-
-2. **Room**
+1. **Room**
 
 	Room is a library that provides an abstraction layer over the SQLite database. It supports all SQL functionalities.
 
 	More info with code examples is available [here](https://developer.android.com/topic/libraries/architecture/room.html).
 
 	What's important about this library are migrations. They are crucial if you wish to migrate your database to a new version. If you don't set up migrations, the room will not crash, nor will it log anything. What it will do is **delete all database data and rebuild the database.** You do **not** want this to happen in production, so be careful when using it and set up your migrations properly. If you are not sure how to do it, ask your team lead to help you out.
 
-3. **DbInspector**
+2. **DbInspector**
```

Comment:
This reminded me - would it be a good idea to open our "In-house libraries" page to the public? Then we could remove them from this page and have them all neatly arranged in a single page for bragging rights. I'd like to hear your opinion on this 🙂

Reply:
I don't see why not :). I think we would benefit from advertising our in-house libs. It was one of the company's pros for me when I was applying :D.